### DISCUSSION TITLE
Assuming you know nothing, where should you begin with Mojo?

### DISCUSSION THREAD

**Discussion by MojoBoingo** [modular/modular#346] [Q&A]
I swear I'm not just interested because it's in my handle.

Like the entire planet I've ping ponged between enamored and a bit wary of LLMs and all the AI things that have and will continue to happen. I'm a self-study student; focusing mostly on a code camp right now and planning to dedicate the next two months to CS50.

So far my focus has been on responsive web design and Git/GitHub. Python was always on the list, and Mojo just makes me want to skip ahead to it. I don't have any background in the languages Mojo is based on (yet), yet I'm intensely curious. Not to mention I applied for and was granted access to the playground.

So this is sort of an open-ended question, no wrong answers; What would anyone recommend for someone with this level of background do if they want to be involved? Right now my best plan is follow the news and the repo and revisit when I've acquired some of that aforementioned background.

**Comment by mojodojodev** [comment]
Hi @MojoBoingo this might help out: https://mojodojo.dev/is_mojo_for_me.html

There isn't any learning resources for Mojo beginners yet, but Python is a fantastic language with tonnes of learning resources, and all the knowledge will transfer over when you're ready to start optimizing at a lower level in Mojo.

**Reply by MojoBoingo** [reply]
More resources are never not helpful, thanks. And yeah that pretty much was my understanding; like I mentioned above I have this itch to move on to Python now but I have loose mental roadmap I'm trying to stick to. I'm going to leave the topic unanswered for now, just to see if anyone else wants to chime in.

I'd ask if you have any openings for someone to pair "Mojo" with catchy words, but I see by your handle that's well in hand.

**Comment by Viddesh1** [comment]
If I know nothing and want to understand the mojo programming language or any other language in general. Here is what I will do:-

- Read through the official documentation.
- Gather as much resources for learning a particular language.
- Practice and implement projects.
- Dwell into the source code.
- Take help from the community.
